feebleline.el

```diff
@@ -72,47 +85,41 @@
   (if feebleline-mode
       (progn
         (setq feebleline/mode-line-format-default mode-line-format)
-        (setq feebleline/timer (run-with-timer 0 0.1 'mode-line-proxy-fn))
+        (setq feebleline/timer (run-with-timer 0 0.1 'feebleline-mode-line-proxy-fn))
```

Comment:
You had several functions/variables without `feebleline` as a prefix. I changed that because these prefixes are Emacs' substitute for namespaces.